randovania/data/json_data/prime1.txt

```diff
@@ -3322,16 +3391,22 @@ Asset id: 3734860277
   * Plasma Door to Fungal Hall A/Dock to Fungal Hall Access
   > Dock to Elevator B
       Space Jump Boots
+  > Pickup (Missile)
+      Morph Ball and Phazon Damage ≥ 100
```

Comment:
Might be worth having a suit requirement here as the no trick path, and without the suit as beginner movement or something

Reply:
Fixed